packages/components/organisms/f-takeawaypay-activation/src/components/ActivationNotLoggedIn.vue

```diff
@@ -0,0 +1,76 @@
+<template>
+    <div>
+        <bag-celebrate-bg-icon :class="$style['c-takeawaypayActivation-icon']" />
+
+        <p :class="$style['c-takeawaypayActivation-title']">
```

Comment:
Can this be a semantic tag to indicate a title, please?

Reply:
@benwhite-deltas Updated to h1.

Reply:
I don't have enough context for this component but are you sure that h1 is the correct title tag in this case. Will this component be used on payment page?

Reply:
@dandel10n Sorry for not adding context. This component will be used on the new page in CoreWeb and will be used for TakeawayPay activation flow. 

So title / h1 is marked here. Does it make sense to keep it as h1?

![image](https://user-images.githubusercontent.com/30234676/123793575-89285200-d8ea-11eb-9aa6-6abb3a9d8985.png)

Reply:
Yas, as it is the main title on the page, it make sense